src/secure_random.cr

```diff
@@ -29,6 +71,13 @@ module SecureRandom
     slice
   end
 
+  # Generates a UUID (Universally Unique Identifier)
+  # 
+  # It generates a random UUID v4. The returned string contains hexadecimal characters from 00 to ff.
+  # 
+  # ```crystal
+  # SecureRandom.uuid #=> a4e319dd-a778-4a51-804e-66a07bc63358
+  # ```
```

Comment:
Might be worth mentioning that neither sequence stability or nor host uniqueness is guaranteed in this format.